### PR TITLE
chore: Migrate tests to RTL (1)

### LINKED ISSUE #7019: Migrate tests to RTL (1)
### Component test suites to migrate:
-  CodeEditor
-  AboutModal
-  Accordion
-  Alert
-  AlertActionCloseButton
-  AlertGroup
-  ApplicationLauncherSeparator
-  Button
-  Card
-  CardBody
-  CardFooter
-  CardHeader
-  CardHeaderMain
-  CardHeadMain
-  CardTitle
-  Checkbox

### PR DESCRIPTION
**Pull request by jeffpuzzo**
**What**: Closes #7019

### CODE REVIEW

**Review by wise-king-sullyman — COMMENTED** [review]
For the goal of getting us off Enzyme and on RTL this looks good to me.

Does look like some merge conflicts need to be fixed though, once those are done I'm ready to approve.

**Review by tlabaj — APPROVED** [review]
LGTM!  We can merge once conflicts are resolved.

**Pull request merged**
